installer/conf/td-agent-bit.conf

```diff
@@ -0,0 +1,53 @@
+[SERVICE]
+    Flush         1
+    Log_Level     info
+    Daemon        off
+    Parsers_File  parsers.conf
+    Log_File      /var/log/fluent-bit.log
+    HTTP_Server   On

```

Comment:
remove all HTTP_ settings (used for internal monitoring of FB, unless our liveliness probe will somehow rely on it)